docs/eigenlayer/security/withdrawal-delay.md

```diff
@@ -4,7 +4,9 @@ sidebar_position: 5
 
 # Withdrawal Delay
 
-EigenLayer contracts feature a **7-day** withdrawal delay, a critical security measure for instances of vulnerability disclosure or when anomalous behavior is detected by monitoring systems. This delay not only serves as a preventive mechanism but also allows, in certain cases, to help mitigate protocol hacks. When contracts are paused and withdrawals disabled, the system enables arbitrary state or code changes to the contracts through upgrades. While technically feasible, such interventions are not a routine practice and should be approached with caution.
+EigenLayer contracts feature a **7-day** withdrawal delay for LST tokens and Native Restaking, a critical security measure for instances of vulnerability disclosure or when anomalous behavior is detected by monitoring systems. The withdrawal window for EIGEN token is **24-days**, which is designed specifically to support future planned functionality unique to the token.
```

Comment:
The 24 day specification here doesn't seem to add anything to the security context. Should this be moved to some other location?

Reply:
Good point. I like the goal of limiting repeated information. We can keep the specific dates in one place under docs/eigenlayer/restaking-guides/0-restaking-user-guide/README.md

+ @ropcode should we clarify that we use the terms Withdrawal Delay and Escrow Period interchangeably? Do they have exactly the same meaning for our protocol's purposes?

Reply:
Resolved now